### PR TITLE
Document Asset Management

### PR DESCRIPTION
**Pull request by AlexMihov**

### CODE REVIEW

**Comment by munen** [comment]
@gabrielhase Can you review this PR, please?

File: reference-docs/server-configuration/config.md [hunk 1]
```diff
@@ -451,6 +506,62 @@ push_notifications: {
 
 For push notifications to be enabled you also need to follow the steps in the [channel config setup](./channel-config.md#push-notifications)
 
+
+#### Asset management
```
Comment (gabrielhase) [inline reference-docs/server-configuration/config.md]: why is this twice in the same file?

**Comment by munen** [comment]
@gabrielhase After looking through the comments and linked PRs, we made the changes that you requested. This PR is ready for review, again.

File: reference-docs/editor-configuration/editing-features.md [hunk 2]
```diff
@@ -291,3 +291,56 @@ comments: {
...
+
+### Upload
+
+In the editor, you can define how many images can be uploaded at the
+same time.
+
+```
+{
+  imageUpload: {
+    maxNumberOfFiles: 10
```
Comment (gabrielhase) [inline reference-docs/editor-configuration/editing-features.md]: do you have a recommendation for this setting? i.e. will the asset management crash if we upload 100 images at the same time?

Comment (munen) [reply reference-docs/editor-configuration/editing-features.md]: @gabrielhase You want a 100 images, I raise to 120: https://www.dropbox.com/s/6z4e305sp4tjj27/demo_120_images_at_once.gif?dl=1

And that on a slow 30mbit connection.

\(^-^)/

Two things:

1. jQuery File Upload automatically batches the uploads in bigger and bigger batches (similar to TCP)
2. During the tests we realized that the max Limit is not really enforced. That's either a bug in the jQuery File Upload or a misconfiguration on our part. We couldn't fix it right away and opted to write a separate issue for that which we'll add to the DAM epic if that's alright with you.

**Pull request merged**